BookingsApi/BookingsApi.IntegrationTests/Api/V1/Hearings/CloneHearingTests.cs

```diff
@@ -18,11 +19,11 @@ public class CloneHearingTests : ApiTest
     public async Task should_return_all_cloned_hearings_for_the_dates_with_unspecified_duration()
     {
         // arrange
-        var startingDate = DateTime.UtcNow.AddMinutes(5);
+        var startingDate = DateTime.UtcNow.AddHours(1);
```

Comment:
Hi just wondering about the start time, does it not need to be within 30 minutes to test the scenario?

Reply:
I added a separate test for that scenario:

```
[Test]
public async Task should_clone_hearing_scheduled_within_30_minutes_from_now()
```

That way we have coverage for times both inside and outside the 30 min window.

Reply:
Ah ok I can see there is another test for that specifically